.travis.yml

```diff
@@ -40,58 +41,68 @@ matrix:
 
         # Try MacOS X
         - os: osx
-          env: PYTHON_VERSION=2.7 SETUP_CMD='test' OPTIONAL_DEPS=true
+          env: PYTHON_VERSION=2.7 SETUP_CMD='test'
 
         # Check for sphinx doc build warnings - we do this first because it
-        # runs for a long time
+        # runs for a long time. WCSAxes is a dependency for the docs.
         - os: linux
-          env: PYTHON_VERSION=2.7 SETUP_CMD='build_sphinx -w'  OPTIONAL_DEPS=true
-          # OPTIONAL_DEPS needed because the plot_directive in sphinx needs them
+          env: PYTHON_VERSION=2.7 SETUP_CMD='build_sphinx -w'
+               PIP_DEPENDENCIES='beautifulsoup4 wcsaxes'
```

Comment:
- indeed, you're right with ``beautiful-soup``, it's not available with 3.5 on conda, but at the end we don't test the optional dependencies with it.

- wcsaxes is only needed for the docs, so it works as is now.